[firebase_admob] update AGP, gradle and admob (#1942)
flutter/plugins: packages/firebase_admob/CHANGELOG.md

## 0.9.0+4

> * Android : Update AGP, gradle and admob.

Reviewer: Can you include the warning removal here in the changelog?

## 0.9.0+4

> * Update AGP, gradle and admob.

Reviewer: ```suggestion
* Update Android gradle plugin, gradle, and Admob versions.
```

> * Fixes warnings about possible null pointer exception in the Android plugin.

Reviewer: ```suggestion
* Improvements to the Android implementation, fixing warnings about a possible null pointer exception.
```
